charts/mungegithub/chart/values.kubernetes.kubernetes.yaml

```diff
@@ -34,7 +34,6 @@ config:
     pull-kubernetes-bazel-build,\
```

Comment:
@cjwagner what is this file being used for? I think this may need to be updated for some other jobs unrelated to this PR (pull-kubernetes-e2e-gce)

Reply:
These files don't really work anymore now that we have submit-queue and misc-munger deployments separated.
They were supposed to parallel the `configmap.yaml` files so that mungegithub can be deployed with Helm, but we never ended up switching to Helm because we decided to go with Tide instead of managing a mungegithub instance for every kubernetes repo.
/hold cancel